sdk/security/keyvault/azadmin/backup/utils_test.go

```diff
@@ -55,25 +53,22 @@ func run(m *testing.M) int {
 		}()
 	}
 
-	if recording.GetRecordMode() == recording.PlaybackMode {
-		credential = &FakeCredential{}
-	} else {
-		tenantID := lookupEnvVar("AZADMIN_TENANT_ID")
-		clientID := lookupEnvVar("AZADMIN_CLIENT_ID")
-		secret := lookupEnvVar("AZADMIN_CLIENT_SECRET")
-		var err error
-		credential, err = azidentity.NewClientSecretCredential(tenantID, clientID, secret, nil)
-		if err != nil {
-			panic(err)
-		}
+	var err error
+	credential, err = azcred.New(nil)
+	if err != nil {
+		panic(err)
 	}
 
-	hsmURL = getEnvVar("AZURE_MANAGEDHSM_URL", fakeHsmURL)
-	blobURL = getEnvVar("BLOB_CONTAINER_URL", fakeBlobURL)
-	token = getEnvVar("BLOB_STORAGE_SAS_TOKEN", fakeToken)
+	hsmURL = recording.GetEnvVariable("AZURE_MANAGEDHSM_URL", fakeHsmURL)
```

Comment:
Just an FYI style point, you can do this when you declare these variables above at package scope:
```go
var (
	blobURL = recording.GetEnvVariable("BLOB_CONTAINER_URL", fakeURL)
	hsmURL = recording.GetEnvVariable("AZURE_MANAGEDHSM_URL", fakeHsmURL)
	sasToken = recording.GetEnvVariable("BLOB_STORAGE_SAS_TOKEN", recording.SanitizedValue)
)
```
You could also inline the fake URLs if you don't need to reference them anywhere else

Reply:
fixed in 2b6cf72